resources/lang/en/book.php

```diff
@@ -26,5 +26,7 @@
     'confirm' => [
         'title' => 'Confirm',
         'delete' => 'Do you want to delete this book'
-    ]
+    ],
+    'common' => 'Sorry, Not Be Found.'
```

Comment:
hiểu nhầm ý mình rồi. 
Luân thấy common với Sorry, Not Be Found. có liên quan với nhau không? không đúng không?
đổi lại đi. message => ['not_found' => 'Sorry, Not be found']  chẳng hạn